lib/cli/commands/deploy.ts

```diff
@@ -45,7 +45,6 @@ export const handler = async (args: DeployArgs) => {
   })
 }
 
-// TODO this verification should be executed when deploying as well
```

Comment:
This function is already called in the deploy handler in `lib/cli/commands/deploy` I guess somone forgot to remove it